classList.go

```diff
@@ -0,0 +1,15 @@
+package dom
+
+import "github.com/dennwc/dom/js"
+
+type ClassList struct {
```

Comment:
Also, please implement the `As***` function, so users can wrap other values into this type.